src/renderer/layer/tilelayer/TileLayerCanvasRenderer.js

```diff
@@ -406,6 +406,13 @@ class TileLayerCanvasRenderer extends CanvasRenderer {
             return;
         }
         if (tileImage instanceof Image) {
+            tileImage.onerrorTick = tileImage.onerrorTick || 0;
```

Comment:
Retry can also be applied when tileImage is not an Image.
You can move the code out of this if block.
```js
tileImage.onerrorTick = tileImage.onerrorTick || 0;
//...
if (tileImage instanceof Image) {
  this.abortTileLoading(tileImage);
}
```